www/kubl05/SP1/about.html

```diff
@@ -20,7 +20,9 @@
     <div class="wrapper">
         <div class="container-top">
             <header>
-                <a href="index.html" class="title">Lukáš Kubík</a><br>
+                <a href="index.html"><img class="logo" src="img/logo.png" onmouseover="this.src='img/logo-hover.png';"
```

Comment:
tak tohle vidim snad poprve v zivote

Reply:
the force of _js_ awakens

Reply:
resil bych to spis pomoci CSS, tj. background-image a :hover, `<img>` tag sam nepouzivam uz dlouho

Reply:
Vzhledem k tomu, že mě ten efekt napadl až poté, co jsem tam měl img, tak jsem použil tohle.. ale oki, chápu, popřemýšlím o tom ještě..